src/vscode/configuration/PowerBIConfiguration.ts

```diff
@@ -43,8 +43,8 @@ const CLOUD_CONFIGS: { [key: string]: iCloudConfig } = {
 	},
 	"USGovCloud": {
 		"friendlyName": "US Government Community Cloud (GCC)",
-		"authenticationProvider": "microsoft-sovereign-cloud",
-		"authenticationEndpoint": "https://login.microsoftonline.com/common",
+		"authenticationProvider": "microsoft",
```

Comment:
according to ``POST https://api.powerbi.com/powerbi/globalservice/v201606/environments/discover?client=powerbi-msolap` the authentication endpoint should have been correct - interesting that it was not working (I dont have any GCC or sovereign cloud access at all so I cant test)

the `authenticationProvider` is used to drive VSCodes internal Microsoft authentication provider and in case of `microsoft-sovereign-cloud` we also set the VSCode setting `microsoft-sovereign-cloud.endpoint` to `<authenticationEndpoint>`

anyway, if you have tested it on your side and it works thats fine for me as I cannot test at all